src/main/java/com/github/dadrus/App.java

```diff
@@ -6,12 +6,17 @@
  */
 public class App 
 {
+	public static final String PASSWORD="boo";
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Remove this hard-coded password. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2068)